Source/WebKit/WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf.cpp

```diff
@@ -277,7 +283,95 @@ AcceleratedSurfaceDMABuf::RenderTargetEGLImage::RenderTargetEGLImage(uint64_t su
 
     WebProcess::singleton().parentProcessConnection()->send(Messages::AcceleratedBackingStoreDMABuf::DidCreateBuffer(m_id, size, format, WTFMove(fds), WTFMove(offsets), WTFMove(strides), modifier, usage), surfaceID);
 }
+#endif // USE(GBM)
+
+#if OS(ANDROID)
+// TODO: There is another copy of this function in GLDisplay.cpp, remove one.
```

Comment:
A common `AHardwareBufferUtilities.h`?

Reply:
Yes, as I outlined in a previous comment.